### PR TITLE
GDPAPPS-400: recommendations, profiles, topics

### PR DESCRIPTION
**Pull request by mhockenbury**
General and user specific recommendations working `GET /recs/<userId>`
User profile storage and retrieval implemented `GET /profile/<userId>`
User topics listing implemented `GET /profile/<userId>/topics`
user topics updating implemented `POST /profile/<userId>/topics`

### CODE REVIEW

File: appserv_akira/application.py [hunk 1]
```diff
@@ -218,4 +248,42 @@ def get(self):
...
+    def get(self, userId):
+        """ Get a list of trending assets """
+        # payload = {"fields":["events"], "start": "-30d", "filters":[{"type":"user", "group":"gci", "item":"ca91516f-9188-4bcd-90cc-b8bbdcc4d7f8"}]}
+        payload = {"fields":["events"], "start": "-7d", "filters":[{"type":"user", "group":"gci", "item":"ca91516f-9188-4bcd-90cc-b8bbdcc4d7f8"}], "historyFields": ["events"], "historyBuckets":7}
+        traffic = network.make_post_request(cfg.get('CXENSE').get('api_traffic'), payload, make_cxense_header("appservices@gannett.com"))
+        return jsonify(traffic)
+
+
+# DNI
+@api.route('/trending', endpoint='trending')
```
Comment (alon85) [inline appserv_akira/application.py]: My mistake - let's change "trending" to "recent"

File: appserv_akira/application.py [hunk 2]
```diff
@@ -60,12 +68,13 @@ def get_generic_recommendations():
...
     else:
+        generic_recs = ['asset_' + item.get('recs-articleid') for item in generic_recs.get('assets').get('items') if item.get('recs-articleid')]
         return database.get_multi_cb(generic_recs.get('assets'), 'core')
 
 
 def create_akira_feed(recommendations):
     # TODO:  base hardcoded indices on config values
-    recommendations = recommendations['items']
+    recommendations = ['asset_' + item.get('recs-articleid') for item in recommendations.get('items') if item.get('recs-articleid')]
     personal_stories = recommendations[0:5] + [recommendations[10]]
```
Comment (alon85) [inline appserv_akira/application.py]: just a note to pay attention to the "TODO" item.

File: appserv_akira/application.py (hunk 1 above)
Comment (mhockenbury) [inline appserv_akira/application.py]: Copy that. updating accordingly

File: appserv_akira/application.py (hunk 2 above)
Comment (mhockenbury) [inline appserv_akira/application.py]: Noted. I plan to break this up in a seperate PR.

**Pull request merged**